File: LeaveManagementSystem/src/main/java/com/adarsh/LeaveManagementSystem/refDto/Employee.java
package com.adarsh.LeaveManagementSystem.refDto;

import java.time.LocalDate;

import lombok.Data;
@Data
public class Employee {

        private Long id;
        private String firstName;
        private String lastName;
        private String email;
        private String phone;
        private String department;
        private String position;
        private LocalDate hireDate;
        private String roletype;//employee/hr/manager



}

File: cmsfrontend1.1/src/app/dto/Employee.ts
export interface Employee {
  // image: null;
  id: number;
  empid:number;
  firstName: string;
  lastName: string;
  email: string;
  phone: string;
  department: string;
  position: string;
  hireDate: Date;
  roletype: string; // 'employee' | 'hr' | 'manager'
  gender: string;
  address: string;
  image: string | null;
}